cpp/src/io/orc/stats_enc.cu

```diff
@@ -378,7 +378,7 @@ __global__ void __launch_bounds__(encode_threads_per_block)
         //  optional sint64 maximumUtc = 4;
         // }
         if (s->chunk.has_minmax) {
-          cur[0] = 7 * 8 + PB_TYPE_FIXEDLEN;
+          cur[0] = 9 * 8 + PB_TYPE_FIXEDLEN;
```

Comment:
What are 7 and 9 here?

Reply:
This is the protobuf field number as described in the comment just above this:
https://github.com/rapidsai/cudf/blob/ff022e6c1e3fbb75974c0b34a56129b04fffef13/cpp%2Fsrc%2Fio%2Forc%2Fstats_enc.cu#L351-L359
It was wrong for this field. should've been 9 (timestampStatistics) but was 7 (dateStatistics)

Similar comments describe the protobuf field number for other fields:
https://github.com/rapidsai/cudf/blob/ff022e6c1e3fbb75974c0b34a56129b04fffef13/cpp%2Fsrc%2Fio%2Forc%2Fstats_enc.cu#L314-L319
https://github.com/rapidsai/cudf/blob/ff022e6c1e3fbb75974c0b34a56129b04fffef13/cpp%2Fsrc%2Fio%2Forc%2Fstats_enc.cu#L278-L285

Magic numbers are bad so this should be addressed in another PR.